### ISSUE TITLE
Meta-issue for work on plugins

### ISSUE THREAD

**Issue by juliendelplanque** [juliendelplanque/DrTests#17]
Need to make plugins system abstract enough to be able to plug various analysis on tests.

- #4 
- #5 
- #16 
- #15

**Comment by juliendelplanque** [comment]
The plugins should be useable without the need to instantiate the UI.

Something like:

```
DTTestsRunnerPlugin new
     configuration: aTestConfiguration;
     run
```

We will need to discuss what are the requirements for the test configuration object.

We will also need to discuss what kind of objects a plugin returns a results of its execution.

**Comment by juliendelplanque** [comment]
This issue is kind-of fixed (except #15) but we do not use meta-issue anyway so let's close it.
